doc/relevancy.md

```diff
@@ -0,0 +1,383 @@
+
+### Debugging es_score
+
+If you want to understand why a result has a given `es_score`, you can use the elasticsearch [Explain API](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-explain.html). This is exposed by the Search API.
+
+You can see the reasons behind an `es_score` by including the `debug=explain` query parameter in your query.
+This will add an `_explanation` field to every result, similar to a SQL-like [EXPLAIN](https://wiki.postgresql.org/images/4/45/Explaining_EXPLAIN.pdf) but a little easier to understand.
+
+Example: https://www.gov.uk/api/search.json?debug=explain&q=harry%20potter
+
+The query params `debug=show_query&count=0` are also sometimes handy.
+
+## Impacts on relevancy
+
+Out of the box, elasticsearch comes with a decent scoring algorithm. They have an excellent [guide on scoring relevancy](https://www.elastic.co/guide/en/elasticsearch/guide/master/scoring-theory.html) which is worth reading.
+
+We've done some work in Search API to [tune relevancy](https://www.elastic.co/guide/en/elasticsearch/guide/master/relevance-conclusion.html), which we go into below.
+
+'Tuning' essentially means overriding elasticsearch defaults for our purposes.
+
+### Relevancy impact contents
+
+1. Boosting
+2. Best bets
+3. Ignoring stopwords
+4. Synonyms
+5. Analyzer usage
+
+### Boosting
+
+We don't only use the query relevancy score to rank documents, we
+apply some additional boosting factors:
+
+- **Popularity:** more visited documents are likely to be more useful
+  to users.
+- **Recency:** a more recently updated document is more likely to be
+  useful than an older document.
+- **Properties:** we have domain-specific knowledge about our
+  content, and can make judgements on what sorts of content are more
+  likely to be useful to the average user.
+
+#### Popularity
+
+This is updated nightly from recent Google Analytics data, gathered by
+the [search-analytics][] application.
+
+Here is the initial reasoning for the numbers (May 2014)
+([6cbd84f][]):
+
+> This field is populated from the page-traffic index, which in turn
+> is populated from a dump file produced by the python scripts in the
+> search-analytics repository.
+
+> The popularity is currently defined here as 1.0 / the rank of the
+> page in terms of traffic.  ie, Our highest traffic page gets 1.0,
+> the next highest gets 0.5, the next highest gets 0.333.  This is a
+> nice and stable measure.
+
+If we want to boost the popularity of a query then we do so by 0.001
+(`POPULARITY_OFFSET = 0.001` in [popularity.rb][]). It is not clear
+where this number came from.
+
+We also boost using `boost_mode: :multiply` (multiply the `_score`
+with the function result (default)).
+
+The rank of a page relative to other pages on GOV.UK is stored in its
+`rank_14` field.  At one point, its number of page views was stored in
+the `vc_14` field, which could have been used to allow us to perform
```

Comment:
We actually still do have `vc_14` (though it's called `vf_14`) - these fields are in the `page-traffic` index, not in the search indices.